docs/site/BelongsTo-relation.md

```diff
@@ -188,7 +188,7 @@ export class OrderController {
   async getCustomer(
     @param.path.number('id') orderId: typeof Order.prototype.id,
   ): Promise<Customer> {
-    return await this.orderRepository.customer(orderId);
+    return this.orderRepository.customer(orderId);
```

Comment:
Now no `await` is used. Do we want to remove `async` keyword for the method signature?

Reply:
Maybe keep `async` so that the code can be modified to use `await` before the `return` statement.